lib/logstash/inputs/beats.rb

```diff
@@ -217,7 +217,7 @@ def start_buffer_broker
         while !stop?
           @output_queue << @buffered_queue.take
         end
-      rescue InterruptionException => e
+      rescue java.lang.InterruptedException => e
         # If we are shutting down without waiting the queue to unblock
         # we will get an `InterruptionException` in that context we will not log it.
```

Comment:
I will update the documentation to make it clear thats not a ruby exception but a java one.

I don't think there is a reliable way to test this, this issues was raised by the current tests, but I never got that locally..


I am not sure if its worth it to use a stub the `#take` call to raise that exception? This seems fragile to me.